src/client/src/App.js

```diff
@@ -0,0 +1,23 @@
+import './App.css';
+const React = require('react')
+const HomePage = require('./components/homepage')
+const Register = require('./components/register')
+const ChatPage = require('./components/chatpage')
+const {BrowserRouter, Route} = require('react-router-dom')
+
+
+class App extends React.Component {
+  render() {
+    return (
+      <BrowserRouter>
+        <div className='container-fluid'>
+          <Route exact path='/' component={HomePage} />
+          <Route path='/register' component={Register} />
```

Comment:
What about keeping those paths in some file, and just importing them everywhere?

Reply:
Right now. What about the future?

Reply:
Then I will extract them in the future, premature abstraction is the root of all evil.

See source - https://news.ycombinator.com/item?id=11047054